.changeset/yellow-doors-raise.md

```diff
@@ -0,0 +1,5 @@
+---
+"@cmpsr/components": patch
+---
+
+Add a new token and a new prop to progressBar
```

Comment:
nit: we should make explicit the new prop that has been added and to which component